sf-docker/all-in-one/docker-compose.yml

```diff
@@ -535,25 +554,6 @@ services:
     volumes:
       - "./iengine.lic:/etc/innovatrics/iengine.lic"
 
-  object-detector:
```

Comment:
is this deletion intentional? do we not want to deploy it in SF yet?

Reply:
no I have reorg. it